ArabicToRomanNumbersChanger.java

```diff
@@ -0,0 +1,131 @@
+public class ArabicToRomanNumbersChanger {
+    private StringBuilder RomanString = new StringBuilder();
+    private String romanThousand = "M";
```

Comment:
please Introduce enum for each roman number. It is much better than use it as string